src/WorkItems/Logging/MetricsLogValues.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.WorkItems.Logging
+{
+    /// <summary>
+    /// MetricsLogValues is variation of FormattedLogValues that accepts a dictionary to store customDimensions Application Insights.
```

Comment:
>customDimensions Application Insights [](start = 95, length = 37)

Comment needs a little grammar/other attention.

Reply:
fixed

---
In reply to: [396834511](https://github.com/microsoft/sarif-sdk/pull/1820#discussion_r396834511) [](ancestors = 396834511)